### ISSUE TITLE
Make "auto-run" checkbox more noticeable.

### ISSUE THREAD

**Issue by bennadel** [bennadel/TinyTest#2]
Right now, the auto-run checkbox is really small and out of the way. Let's make this more noticeable so people will know to use it.

**Comment by danshort** [comment]
I agree, and if you like this I'll commit it (along with the changes to the LESS file).
![autorunnoclicky](https://f.cloud.github.com/assets/44993/1409275/210af446-3d90-11e3-8068-f4fd022b11d7.png)
![autorunyesclick](https://f.cloud.github.com/assets/44993/1409278/306cbff0-3d90-11e3-8991-0462b16344e9.png)
